types/browser-image-compression/index.d.ts

```diff
@@ -0,0 +1,25 @@
+// Type definitions for browser-image-compression 1.0
+// Project: https://github.com/Donaldcwl/browser-image-compression
+// Definitions by: Donald <https://github.com/Donaldcwl>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface Options {
+    /** @default Number.POSITIVE_INFINITY */
+    maxSizeMB?: number;
+    /** @default undefined */
+    maxWidthOrHeight?: number;
+    /** @default false */
+    useWebWorker?: boolean;
+    /** @default 10 */
+    maxIteration?: number;
+    /** Default to be the exif orientation from the image file */
+    exifOrientation?: number;
+    /** A function takes one progress argument (progress from 0 to 100) */
+    onProgress?: (progress: number) => void;
+    /** Default to be the original mime type from the image file */
+    fileType?: string;
+}
+
+declare function imageCompression(image: File | Blob, options: Options): Promise<File | Blob>;
+
+export = imageCompression;
```

Comment:
```suggestion
declare function imageCompression(image: File | Blob, options: Options): Promise<File | Blob>;

export as namespace imageCompression;

export = imageCompression;
```
what about the browser?
I would like to use sample code stright from the readme;
```ts
async function handleImageUpload({ target }: { target: EventTarget & { files: FileList } }) {
    const imageFile = target.files[0];
    console.log('originalFile instanceof Blob', imageFile instanceof Blob); // true
    console.log(`originalFile size ${imageFile.size / 1024 / 1024} MB`);

    const options = {
        maxSizeMB: 1,
        maxWidthOrHeight: 1920,
        useWebWorker: true,
    };
    try {
        const compressedFile = await imageCompression(imageFile, options);
        console.log('compressedFile instanceof Blob', compressedFile instanceof Blob); // true
        console.log(`compressedFile size ${compressedFile.size / 1024 / 1024} MB`); // smaller than maxSizeMB

        // await uploadToServer(compressedFile);
    } catch (error) {
        console.log(error);
    }
}
```

Reply:
Thank you.